android/Reviewer/app/src/main/java/rs/reviewer/activities/SplashScreenActivity.java

```diff
@@ -15,11 +14,11 @@ protected void onCreate(Bundle savedInstanceState)
     {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.splash);
-        int SPLASH_TIME_OUT = 3000;
+        int SPLASH_TIME_OUT = 2000;
         new Timer().schedule(new TimerTask() {
             @Override
             public void run() {
-                startActivity(new Intent(SplashScreenActivity.this, MainActivity.class));
+                startActivity(new Intent(SplashScreenActivity.this, LoginActivity.class));
```

Comment:
ovde treba provera da li ima korisnika logovanog ili ne (na osnovu toga sto cuvas u sqlite bazi - i na logout bi trebalo da se brise iz baze)

Ako je korisnik ulogovan pokazuje mu se MainActivity, u suprotnom LoginActivity

Reply:
dodao log out dugme i sredio to za splash scrren